server/plugins/Essentials/config.yml

```diff
@@ -326,7 +326,7 @@ player-commands:
 # This is useful if you want superperms-based permissions (with wildcards) for custom permissions plugins.
 # If you wish to use EssentialsX's built-in permissions using the `player-commands` section above, set this to false.
 # Default is true.
-use-bukkit-permissions: true
+use-bukkit-permissions: false
```

Comment:
So this will tell essentials to use the other permissions system set up instead of the hard coded list in this file?

Reply:
No, it wasn't using the permissions in the file before. And because we don't have a permissions plugin, all the commands were defacto disabled. I changed it to use the ones in file, and we can switch it back if we ever use another permissions plugin.

Reply:
Ahh, I thought there was a permissions plugin though. 

Isn't vault supposed to handle permissions?

Reply:
As far as I knew vault just provides a compatibility layer but doesn't handle anything itself. That might have changed since the last time I dealt with vault.